### PR TITLE
Using Github Actions to deploy to arcs-live.

### PR DESCRIPTION
**Pull request by alxrsngrtn**
- Added a quick Github Action to fix the arcs-live page.
  - Re-uses the deploy script from the Travis CI days.
- Removed the TODO at the top of the README.

CC: @shaper and @Cypher1 (I believe you were looking into GH Actions recently)

### CODE REVIEW

File: .github/workflows/live-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# An Action to build Web Arcs and deploy to https://live.arcs.dev
+name: live-deploy
+
+on:
+  push:
+    branches: [ master ]
```
Comment (bgogul) [inline .github/workflows/live-deploy.yml:6]: Is there a way to make sure only green commits on master are deployed?

**Review by bgogul — COMMENTED** [review]
Thanks for taking care of this, Alex. Do we need to allocate any resources for this in GCB or in other place?

**Comment by alxrsngrtn** [comment]
> Do we need to allocate any resources for this in GCB or in other place?

No, I don't think we need to allocate any other resources outside of what Github offers.

File: .github/workflows/live-deploy.yml (hunk 1 above)
Comment (alxrsngrtn) [reply .github/workflows/live-deploy.yml:6]: It looks like there's an action that will fit nicely here: https://github.com/marketplace/actions/wait-for-check

We can run the deploy step only after GCB passes on master.

Comment (bgogul) [reply .github/workflows/live-deploy.yml:6]: Do you want to make that change before merging this PR in?

Comment (alxrsngrtn) [reply .github/workflows/live-deploy.yml:6]: Yes, I do. 
Having read more documentation, I think a simpler / cleaner approach would be to run the typescript tests + lint in the Action.

File: .github/workflows/live-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run Typescript Tests
+        run: |
```
Comment (bgogul) [inline .github/workflows/live-deploy.yml:16]: Is `|` a typo?

File: .github/workflows/live-deploy.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+          tools/sigh webpack
+          tools/sigh test --all
+
+      - name: Run deployment script
```
Comment (bgogul) [inline .github/workflows/live-deploy.yml:20]: How do you make sure that this step waits for the previous step?

File: .github/workflows/live-deploy.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# An Action to build Web Arcs and deploy to https://live.arcs.dev
```
Comment (bgogul) [inline .github/workflows/live-deploy.yml:1]: Is `.github/workflows/live-deploy.yml` the right directory? Or, should this move to tools?

File: .github/workflows/live-deploy.yml (hunk 1 above)
Comment (bgogul) [reply .github/workflows/live-deploy.yml:6]: I am actually wondering whether this should be done as part of GCB itself to reduce the number of dependencies to other services. However, that might only delay things further. Let us start with this and move to GCB If there are issues.

File: README.md [hunk 5]
```diff
@@ -7,12 +7,6 @@ _Our mission is to create an open platform for smart and ubiquitous computing, w
 
 A hosted version of Arcs is available at https://live.arcs.dev.
 
-> Note: Arcs is transitioning to Cloud Build and the live.arcs.dev hosted pages
```
Comment (bgogul) [inline README.md:10]: How do we know of errors in depolyment? Is there a badge we can add to the home page?

**Review by bgogul — APPROVED** [review]

File: .github/workflows/live-deploy.yml (hunk 2 above)
Comment (alxrsngrtn) [reply .github/workflows/live-deploy.yml:16]: Nope! I need it to specify that the next two lines need to be run as a script.

File: .github/workflows/live-deploy.yml (hunk 4 above)
Comment (alxrsngrtn) [reply .github/workflows/live-deploy.yml:1]: No, it needs to live in this directory. `.github` is a reserved folder for actions + github automation.

File: README.md (hunk 5 above)
Comment (alxrsngrtn) [reply README.md:10]: Yes, and I believe we can get the URL for the badge from the `actions` tab. Let me see if I can get it before we start running the pipelines...

File: .github/workflows/live-deploy.yml (hunk 3 above)
Comment (alxrsngrtn) [reply .github/workflows/live-deploy.yml:20]: Generally, jobs run in parallel, but steps run sequentially [1]. The way this is configured, the deploy script will wait for the build step. 

[1] https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/introduction-to-github-actions#steps

**Comment by alxrsngrtn** [comment]
>  Let us start with this and move to GCB If there are issues.

I think that's a good idea and agree with this plan.

File: README.md (hunk 5 above)
Comment (alxrsngrtn) [reply README.md:10]: It looks like the action has to land before I can generate a badge.

**Pull request merged**